ramls/requests-batch.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Collection of requests",
+  "type": "object",
+  "properties": {
+    "requests": {
+      "description": "List of request items to update",
+      "id": "requests",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "$ref": "request.json"
+      }
+    },
+    "transactionMode": {
+      "type": "string",
+      "description": "Transaction mode to use for update operation: Same - execute all updates sequentially within the same transaction",
+      "enum": [
+        "Same"
+      ],
+      "default": "Same"
+    }
+  },
```

Comment:
What is the transaction mode?

If there is only a single mode, what value does introducing the parameter have?

Reply:
Currently all updates are executed subsequently within the same transaction, this is what transactionMode = same defines. But it might be a case when we wont to execute updates concurrently and within separate transaction (this is how batch API for instances works now). For this we can introduce a new transactionMode. 

I've added this parameter to set up a basic architecture of the resource.

Reply:
> Currently all updates are executed subsequently within the same transaction, this is what transactionMode = same defines. But it might be a case when we wont to execute updates concurrently and within separate transaction (this is how batch API for instances works now). For this we can introduce a new transactionMode.
> 
> I've added this parameter to set up a basic architecture of the resource.

I can understand creating space for these kinds of options, by using an object and having the requests as a property.

What I don't understand is why commit to the details of the parameters at this stage, when they don't offer any choices for the client? 

Maybe it is trying to guide the future changes in a certain way? I'm cautious about that because it can lead to premature commitment. I understand that deferring can sometimes lead to misunderstandings when it is introduced.